CONTRIBUTING.md

```diff
@@ -2,13 +2,12 @@ There are several ways to contribute to Audacity:
 
 ## Developing
 
-Audacity is mostly coded in C++. You'll need to be at least somewhat comfortable in it to start contributing code. 
+Audacity is mostly coded in C++. You'll need to be at least somewhat comfortable in it to start contributing code.
 
-If you want to develop Audacity: 
+If you want to develop Audacity:
```

Comment:
[Nitpick] I think adding the verb “help” would facilitate a more collaborative working environment for this project.
```suggestion
If you want to help develop Audacity:
```